security/compliance_operator/compliance-operator-release-notes.adoc

```diff
@@ -13,6 +13,24 @@ These release notes track the development of the Compliance Operator in the {pro
 
 For an overview of the Compliance Operator, see xref:../../security/compliance_operator/compliance-operator-understanding.adoc#understanding-compliance-operator[Understanding the Compliance Operator].
 
+[id="compliance-operator-release-notes-0-1-59"]
+== OpenShift Compliance Operator 0.1.59
+
+The following advisory is available for the OpenShift Compliance Operator 0.1.59:
+
+* link:https://access.redhat.com/errata/RHBA-2022:8538[RHBA-2022:8538 - OpenShift Compliance Operator bug fix update]
+
+[id="compliance-operator-0-1-59-bug-fixes"]
+=== Bug fixes
+
+* Previously, support for Payment Card Industry Data Security Standard (PCI DSS) and PCI DSS node did not show on `ppc64le`, which left profiles disabled in the architecture. Now, support for PCI DSS and PCI DSS node shows on `ppc64le` and enables all profiles in the architecture. (link:https://issues.redhat.com/browse/OCPBUGS-3252[*OCPBUGS-3252*])
```

Comment:
I like that you defined PCI  DSS, let's keep that! 

We are referring specifically to `ocp4-pci-dss` and `ocp4-pci-dss-node` compliance profiles, so maybe something like: 
"... (PCI DSS) compliance profiles `ocp4-pci-dss` and `ocp4-pci-dss-node` did not show..."

Reply:
Additionally, I think we need to change some language here. "Did not show" is ambiguous. Were they unavailable? Disabled? 

Is this a new feature? I don't see PCI DSS support in 0.1.53 and prior releases ([4.6 and 0.1.53 latest release](https://docs.openshift.com/container-platform/4.6/security/compliance_operator/compliance-operator-supported-profiles.html), for example)

Reply:
Added PCI DSS profile as an enhancement since it's new functionality for different architectures.